README.md

```diff
@@ -86,29 +88,14 @@ For instance, `{1, 2, 3, 5}` sequence becomes `{[1, 4), [5, 6)}`.
 True
```

Comment:
maybe add:
`>> bool(Selection([]))` here to show how the `bool` is behaving.

Reply:
I'm more leaning to removing the bool completely - it just works like the normal containers

Reply:
Yes, fair enough.